src/db/config.ts

```diff
@@ -44,15 +44,17 @@ if (environment == 'production') {
     dbUser = process.env.TEST_DB_USERNAME || 'postgres';
     dbPassword = process.env.TEST_DB_PASSWORD;
   } else {
-    throw new Error(`Invalid NODE_ENV ${environment}, must be one of: [development, test, production]`);
+    throw new Error(
+      `Invalid NODE_ENV ${environment}, must be one of: [development, test, production]`
+    );
   }
 
   const dbDriver: Dialect = 'postgres';
   sequelizeConnection = new Sequelize(dbName, dbUser, dbPassword, {
     host: dbHost,
     dialect: dbDriver,
-    logging: environment == 'development'
+    logging: false, // environment == 'development'
```

Comment:
Is this commented code intentional? Do we want to always remove logging?

Reply:
The SQL logging was getting in the way of my console logs.  I don't mind putting it back now.